src/main/kotlin/nl/tudelft/ewi/se/ciselab/testgenie/services/CoverageVisualisationService.kt

```diff
@@ -39,22 +39,7 @@ class CoverageVisualisationService(private val project: Project) {
         val state = ApplicationManager.getApplication().getService(QuickAccessParametersService::class.java).state
         if (state.showCoverage) {
```

Comment:
You can also remove this code duplication while you are at it? :)

Reply:
Oh wow, yeah even more can be reduced!